extensions/amp-moviads-player/0.1/amp-moviads-player.js

```diff
@@ -0,0 +1,890 @@
+    return false;
+  }
+
+  /** @override */
+  getCurrentTime() {
+    return this.playerData_.currentTime;
+  }
+
+  /** @override */
+  getDuration() {
+    return this.playerData_.duration;
+  }
+
+  /** @override */
+  getPlayedRanges() {
+    return this.playerData_.playedRanges;
+  }
+
+  /** @override */
+  seekTo(unusedTimeSeconds) {
+    this.user().error(TAG, '`seekTo` not supported.');
+  }
+
+  /** @override */
+  mapConfig() {
+    const mapConfIndex = {
+      'movieContenerId': null,
+      'movieSrcs': null,
+      'movieSrc': null,
+      'moviePosterSrc': null,
+      'playerIdConf': null,
+      'masterDomain': null,
+      'maxWidth': null,
+      'maxHeight': null,
+      'watermark': null,
+      'UA': null,
+      'lang': 'pl',
+      'autoplay': false,
+      'scrollOnlyDownPlayer': null,
+      'volume': 0,
+      'volumeOnOver': 0.9,
+      'hideWhenNoAd': false,
+      'hideWhenEnd': false,
+      'scrollPlayer': true,
+      'debug': false,
+      'responsive': true,
+      'functionAfterEnded': null,
+      'videoWall': null,
+      'timeToSaveCloseChoice': 2,
+      'adBlockInfo': false,
+      'langText': {
+        'advertisement': 'Reklama',
+        'Seconds': 'sekund',
+        'close': 'zamknij',
+      },
+      'fixedOnMobile': false,
+    };
+    Object.keys(mapConfIndex).forEach(key => {
+      if (this.configMap_[key] !== undefined) {
+        this.config_[key] = this.configMap_[key];
+      } else if (this.config_[key] === undefined) {
+        this.config_[key] = mapConfIndex[key];
+      }
+    });
+
+    if (
+      this.configMap_['UA'] !== undefined &&
+      this.configMap_['UA'].length > 0
+    ) {
+      this.gaUserUA_ = this.configMap_['UA'];
+    }
+    this.setRandMovieSrc();
+  }
+
+  /** @override */
+  mapExternalConfig() {
+    const mapConfIndex = {
+      'cssPuiblisher': null,
+      'apv': 100,
+      'UA': null,
+      'showLogo': false,
+      'watermark': null,
+      'vastUrl': null,
+      'keyPrivate': null,
+      'adBlockInfo': false,
+    };
+    Object.keys(mapConfIndex).forEach(key => {
+      if (this.configLocal_[key] !== undefined) {
+        this.config_[key] = this.configLocal_[key];
+      }
+    });
+
+    if (
+      this.configLocal_['UA'] !== undefined &&
+      this.configLocal_['UA'].length > 0
+    ) {
+      this.gaMoviadsUA_ = this.configLocal_['UA'];
+    }
+  }
+
+  /** @override */
+  setRandMovieSrc() {
+    if (null !== this.config_.movieSrcs && this.config_.movieSrcs.length > 0) {
+      this.config_['dataSrc'] = this.config_.movieSrcs[
+        Math.floor(Math.random() * this.config_.movieSrcs.length)
+        ];
+    } else {
+      this.config_['dataSrc'] = this.config_.movieSrc;
+    }
+  }
+
+  /** @override */
+  hideWhenEnd() {
+    if (this.config_.hideWhenEnd === true) {
+      this.element.remove();
+    }
+  }
+
+  /** @override */
+  hideWhenNoAd() {
+    if (this.config_.hideWhenNoAd === true) {
+      this.element.remove();
+    }
+  }
```

Comment:
You can't remove elements without user interaction since it causes [Content Layout shift](https://web.dev/cls). Remove.